spec/main_spec.rb

```diff
@@ -0,0 +1,55 @@
+require '../lib/board.rb'
+require '../lib/player.rb'
```

Comment:
[OPTIONAL]
Ought to use `require_relative` instead of `require` for files within a project directory.
Basically: Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory.